temoa-uiuc/data_parser.py

```diff
@@ -203,17 +215,26 @@ def create_dataframe(lines, variable, sector='elc', years=time_horizon):
 
     dataframe = pd.DataFrame(technology_dict)
     dataframe.set_index('Year', inplace=True)
-    if variable == 'Emissions':
+    if variable == 'emissions':
         dataframe['total'] = dataframe.sum(axis=1)
+        if emission is not 'co2eq':
+            dataframe['cumulative'] = dataframe['total'].cumsum()
 
     return dataframe
 
 
-def bar_plot(dataframe, variable, scenario, sector, save=True):
+def bar_plot(dataframe, variable, scenario, sector, emission=None, save=True):
     """
     This function creates a bar chart for
     a given dataframe and returns nothing.
 
+    TODO: Currently, this function will plot all technologies
+    in a given system. Even if those technologies do not produce
+    anything (i.e. zero valued). This is evident in the spent-fuel
+    and ewaste plots (which are not stacked) because space is made
```

Comment:
what is 'ewaste' ?

Reply:
"Electronic waste" -- It's a catchall for the solid waste from decomissioning solar panels and wind turbines.